qiskit/providers/ibmq/api_v2/ibmqclient.py

```diff
@@ -0,0 +1,246 @@
+                exclude from the result.
+            included_fields (list[str]): names of the fields to explicitly
+                include in the result.
+
+        Returns:
+            dict: job information.
+        """
+        return self.api_client.job(job_id).get(excluded_fields,
+                                               included_fields)
+
+    def job_status(self, job_id):
+        """Return the status of a job.
+
+        Args:
+            job_id (str): the id of the job.
+
+        Returns:
+            dict: job status.
+        """
+        return self.api_client.job(job_id).status()
+
+    def job_properties(self, job_id):
+        """Return the backend properties of a job.
+
+        Args:
+            job_id (str): the id of the job.
+
+        Returns:
+            dict: backend properties.
+        """
+        # return self.api_client.job(job_id).properties()
+        raise NotImplementedError
+
+    def job_cancel(self, job_id):
+        """Submit a request for cancelling a job.
+
+        Args:
+            job_id (str): the id of the job.
+
+        Returns:
+            dict: job cancellation response.
+        """
+        return self.api_client.job(job_id).cancel()
+
+    # Miscellaneous public functions.
+
+    def api_version(self):
+        """Return the version of the API.
+
+        Returns:
+            dict: versions of the api components.
+        """
+        return self.api_client.version()
+
+    # Endpoints for compatibility with classic IBMQConnector. These functions
+    # are meant to facilitate the transition, and should be removed moving
+    # forward.
+
+    def get_status_job(self, id_job):
+        # pylint: disable=missing-docstring
+        return self.job_status(id_job)
+
+    def run_job(self, qobj, backend_name):
+        # pylint: disable=missing-docstring
+        return self.job_run_qobj(backend_name, qobj)
+
+    def get_jobs(self, limit=10, skip=0, backend=None, only_completed=False,
+                 filter=None):
+        # pylint: disable=missing-docstring,redefined-builtin
+        # TODO: this function seems to be unused currently in IBMQConnector.
+        raise NotImplementedError
+
+    def get_status_jobs(self, limit=10, skip=0, backend=None, filter=None):
+        # pylint: disable=missing-docstring,redefined-builtin
+        if backend:
+            filter = filter or {}
+            filter.update({'backend.name': backend})
+
+        return self.list_jobs_statuses(limit, skip, filter)
+
+    def cancel_job(self, id_job):
+        # pylint: disable=missing-docstring
+        return self.job_cancel(id_job)
+
+    def backend_defaults(self, backend):
+        # pylint: disable=missing-docstring
+        return self.backend_pulse_defaults(backend)
+
+    def available_backends(self):
+        # pylint: disable=missing-docstring
+        return self.list_backends()
+
+    def get_job(self, id_job, exclude_fields=None, include_fields=None):
```

Comment:
This explains the `job_get` name, although I would have preferred a different name.